packages/contract/index.js

```diff
@@ -2,8 +2,8 @@ const Schema = require("@truffle/contract-schema");
 const Contract = require("./lib/contract");
 const truffleContractVersion = require("./package.json").version;
 
-const contract = (json = {}) => {
-  const normalizedArtifactObject = Schema.normalize(json);
+const contract = (json = {}, opts = {}) => {
+  const normalizedArtifactObject = Schema.normalize(json, opts);
```

Comment:
If we're adding an argument to this function, I think we should namespace things (and don't abbreviate `opts`)

```suggestion
const contract = (json = {}, options = {}) => {
  const normalizedArtifactObject = Schema.normalize(json, options.normalize || {});
```